sidebars.json

```diff
@@ -383,8 +383,27 @@
           "type": "doc"
         },
         {
-          "id": "development_suite/set-up-infrastructure/enhanced-project-workflow",
-          "type": "doc"
+          "label": "Enhanced Project Workflow",
+          "type": "category",
+          "collapsed": true,
+          "link": {
+            "id": "development_suite/set-up-infrastructure/enhanced-project-workflow",
```

Comment:
can we rename the file `overview` to `enhanced-project-workflow` so it matches the id? I think it might be clearer

Reply:
enhanced-project-workflow.md is the old file name and I have renamed it in order that it is clearly distinguishable between its subsection files